### PR TITLE
Rails 6.1 upgrade

### LINKED ISSUE #1009: upgrade to rails 6.1
https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-2-to-rails-6-0
https://guides.rubyonrails.org/6_0_release_notes.html
https://guides.rubyonrails.org/active_record_multiple_databases.html

* upgrade to latest Rails 6.1.X
* remove coffee-script
* get rid of second_base in favor of native rails multiple databases
* fix cdr database truncation in tests

### PR DESCRIPTION
**Pull request by senid231**
fixes #1009

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -5,37 +5,36 @@
...
+
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module Yeti
   class Application < Rails::Application
-    config.load_defaults '5.2'
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.0
```
Comment (Fivell) [inline config/application.rb:29]: config.load_defaults 6.1

**Comment by senid231** [comment]
@Fivell @dmitry-sinina done

**Comment by Fivell** [comment]
@senid231  u're  🦸‍♂️

**Comment by gigorok** [comment]
@senid231  👏

**Comment by dmitry-sinina** [comment]
```

**Comment by dmitry-sinina** [comment]
```yeti-web@demo-yeti:~$ RAILS_ENV=production ./bin/bundle.sh exec rake db:migrate
App Timezone Etc/UTC
Yeti DB timezone UTC
CDR DB timezone UTC
ActiveRecord timezone utc
rake aborted!
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class MoveSchemaMigrationsTable < ActiveRecord::Migration[4.2]
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:559:in `inherited'
/opt/yeti-web/db/migrate/20170822151410_move_schema_migrations_table.rb:1:in `<main>'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1045:in `load_migration'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1041:in `migration'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1037:in `disable_ddl_transaction'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1387:in `use_transaction?'
/opt/yeti-web/lib/active_record/migrator.rb:16:in `rescue in block in migrate'
/opt/yeti-web/lib/active_record/migrator.rb:13:in `block in migrate'
/opt/yeti-web/lib/active_record/migrator.rb:10:in `each'
/opt/yeti-web/lib/active_record/migrator.rb:10:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1086:in `up'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1061:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/tasks/database_tasks.rb:237:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:92:in `block (3 levels) in <main>'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:90:in `each'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:90:in `block (2 levels) in <main>'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'

Caused by:
ArgumentError: wrong number of arguments (given 2, expected 1)
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1322:in `execute_migration_in_transaction'
/opt/yeti-web/lib/active_record/migrator.rb:14:in `block in migrate'
/opt/yeti-web/lib/active_record/migrator.rb:10:in `each'
/opt/yeti-web/lib/active_record/migrator.rb:10:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1086:in `up'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/migration.rb:1061:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/tasks/database_tasks.rb:237:in `migrate'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:92:in `block (3 levels) in <main>'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:90:in `each'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/activerecord-6.1.4/lib/active_record/railties/databases.rake:90:in `block (2 levels) in <main>'
/opt/yeti-web/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
yeti-web@demo-yeti:~$ 
```

**Comment by senid231** [comment]
Will fix that.

**Pull request merged**